optimum/exporters/neuron/model_configs.py

```diff
@@ -18,9 +18,22 @@
 from typing import List
 
 from ...utils.normalized_config import NormalizedConfigManager
+from ..tasks import TasksManager
 from .config import TextEncoderNeuronConfig
 
 
+COMMON_TEXT_TASKS = (
+    "default",
+    "masked-lm",
+    "sequence-classification",
+    "multiple-choice",
+    "token-classification",
+    "question-answering",
+)
+register_in_tasks_manager = TasksManager.create_register("neuron")
+
```

Comment:
After talking to @fxmarty, maybe to make this more clear, we could have a `REGISTER` dictionary as follow:

```python
REGISTER = {
    "bert": ("BertTFLiteConfig", COMMON_TEXT_TASKS),
     ...
}

And have a function registering all the classes by reading this dictionary. 
(Also this function should be called in `optimun.exporters.neuron.__init__` to make sure the classes are available?

WDTY?
```

Reply:
I personally dislike that we are taking different directions in subpackages - if this is done this way in optimum-neuron, there is no reason to have `_SUPPORTED_MODEL_TYPE` in optimum. No matter the solution (I think a dict is easier to read and more simple than decorators), I think we should eventually align for all packages.

Reply:
@fxmarty  the registering system is more for "external" packages, "external" meaning not the main optimum package here. For all the rest, using what we currently have is perfectly valid. It is just not the same application.

Reply:
To me it is the same application: listing what is available.